tests/framework/extensions/provisioning/ssh.go

```diff
@@ -8,19 +8,25 @@ import (
 	"errors"
 	"strconv"
 	"strings"
+	"time"
 
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	nodev1 "github.com/rancher/rancher/tests/framework/extensions/nodes"
```

Comment:
```suggestion
	node "github.com/rancher/rancher/tests/framework/extensions/nodes"
```

Reply:
any other suggestion? because I use node in the func 
```
func CallSSHTestByName(testCase SshTestCase, node *nodes.Node, client *rancher.Client, clusterID string) error
```

Reply:
```suggestion
	"github.com/rancher/rancher/tests/framework/extensions/nodes"
```
should be fine then, or `extnodes ` for nodes extensions.